index.html

```diff
@@ -76,5 +101,12 @@
               <input type="text" name="emailaddress">
             </form>
           <p>Enter your information to receive more painful years of Seattle baseball. Wondering how to submit your information? Me too, this will be added later this week.</p>
-  </body>
+
+          <footer><p>Created by Steven WA <br>&copy; 2015</p>
+          </footer>
+
+
+
```

Comment:
There's a bit too much whitespace and indentation here as well. You should set your Sublime default preferences to automatically set those spaces and indents for you if you've not done so already.